connectors-stubs/src/main/ruby/swiftype_exception_tracking.rb

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License;
+# you may not use this file except in compliance with the Elastic License.
+#
+module Swiftype
```

Comment:
Hmmm should we also have `Swiftype` anywhere here?

I'd probably rename it to something else, like `Util::ExceptionTracking`

Reply:
I would too. Unfortunately, then the `connectors_exception_tracking.rb` would not function when dropped back into ent-search.

Reply:
For context, here's where this idea came from: https://github.com/elastic/ent-search/pull/6157#issuecomment-1034138558

Reply:
Can we rename it in `ent-search` as well? I think the Swiftype namespace is shrinking and it's a good opportunity to shrink it a bit more.

Reply:
I think that namespacing is somewhat tied to directory structure in ruby? And there's a lot still in `lib/swiftype`. I'm not opposed to it as a general goal, but I don't think we're going to get a greenlight to prioritize that.
<img width="250" alt="Screen Shot 2022-02-17 at 9 51 53 AM" src="https://user-images.githubusercontent.com/5288246/154518715-d7774562-80e2-4579-9940-144bf1e7118d.png">